pulsar-broker/src/test/java/org/apache/pulsar/client/impl/MultiTopicsReaderTest.java

```diff
@@ -68,7 +69,11 @@ protected void setup() throws Exception {
                 new ClusterData(pulsar.getWebServiceAddress()));
         admin.tenants().createTenant("my-property",
                 new TenantInfo(Sets.newHashSet("appid1", "appid2"), Sets.newHashSet("test")));
-        admin.namespaces().createNamespace("my-property/my-ns", Sets.newHashSet("test"));
+        Policies policies = new Policies();
+        policies.replication_clusters = Sets.newHashSet("test");
```

Comment:
Why need to specify the replication_clusters here?

Reply:
It's just to pass the complete policies object instead of just the replication config as before.